### PR TITLE
Add Course::Event Model

### PR DESCRIPTION
**Pull request by kxmbrian**
Course::Events will be displayed in the lesson plan.

### CODE REVIEW

File: db/migrate/20150615075515_create_course_events.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateCourseEvents < ActiveRecord::Migration
+  def change
+    create_table :course_events do |t|
+      t.string :location
+      t.integer :event_type, default: 0
+
+      t.timestamps null: false
```
Comment (lowjoel) [inline db/migrate/20150615075515_create_course_events.rb]: userstamps?

Comment (kxmbrian) [inline db/migrate/20150615075515_create_course_events.rb]: It is already accounted for since Course::Event already acts_as_lesson_plan_item ?

Comment (lowjoel) [inline db/migrate/20150615075515_create_course_events.rb]: right... then why is there a need for the timestamp?

Comment (kxmbrian) [inline db/migrate/20150615075515_create_course_events.rb]: good point.

**Pull request merged**